code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -808,7 +798,7 @@ butcher
 
 /datum/aiTask/sequence/goalbased/flockdrone_capture/precondition()
 	var/mob/living/critter/flock/F = holder.owner
-	return F?.can_afford(15) && (length(F.flock?.enemies)) //gotta have enemies if we're gonna cage em
```

Comment:
useful comment

Reply:
Maybe would be better as precondition comment?